legal-api/src/legal_api/core/filing.py

```diff
@@ -460,7 +460,10 @@ def _add_ledger_order(filing: FilingStorage, ledger_filing: dict) -> dict:
     @staticmethod
     def _is_display_ledger(filing: FilingStorage) -> bool:
         """Return boolean that display the ledger."""
-        return filing.filing_type != Filing.FilingTypes.ADMIN_FREEZE
+        # If filing is an admin freeze or involuntary dissolution, we will NOT display it on ledger
+        if (filing.filing_type == Filing.FilingTypes.ADMIN_FREEZE or
+            (filing.filing_type == Filing.FilingTypes.DISSOLUTION and filing.filing_sub_type == 'involuntary')): return False
```

Comment:
I changed the logic here so that it's easier to understand and read. I got rid of the negation.

Also, I was hoping to use an ENUM for the `involuntary` but saw that the string was being used everywhere as well. I was thinking of changing it everywhere to use an enum.